modules/installation-creating-aws-vpc-carrier-gw.adoc

```diff
@@ -0,0 +1,105 @@
+endif::[]
+ifeval::["{context}" == "aws-compute-edge-zone-tasks"]
+:post-aws-zones:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-creating-aws-vpc-carrier-gw_{context}"]
+ifdef::wavelength-zone[]
+= Creating a VPC carrier gateway
+endif::wavelength-zone[]
+ifdef::post-aws-zones[]
+= Wavelength Zones only: Creating a VPC carrier gateway
+endif::post-aws-zones[]
+
+To use public subnets in your {product-title} cluster that runs on Wavelength Zones, you must create the carrier gateway and associate the subnet to the VPC. A subnet is useful for deploying load balancers or edge compute nodes.
+
+Carrier gateways exist for VPCs that only contain subnets in a Wavelength Zone. A carrier gateway supports IPv4 traffic.
+
+The following list explains the functions of a carrier gateway in the context of an AWS Wavelength Zones location: 
+
+* Provides connectivity between your Wavelength Zone and the carrier network, which includes any available devices from the carrier network.
+* Performs Network Address Translation (NAT) functions, such as translating IP addresses that are public IP addresses stored in a network border group, from Wavelength Zones to carrier IP addresses. These translation functions apply to inbound and outbound traffic.
+* Authorizes inbound traffic from a carrier network that is located in a specific location.
+* Authorizes outbound traffic to a carrier network and the internet.
+
+[NOTE]
+====
+No inbound connection configuration exists from the internet to a Wavelength Zone through the carrier gateway.
+====
+
+You can use the provided CloudFormation template to create a stack of the following AWS resources:
+
+* One carrier gateway that associates to the VPC ID in the template.
+* One public route table for the Wavelength Zone named as `<ClusterName>-public-carrier`.
+* Default IPv4 route entry in the new route table that targets the carrier gateway.
+* VPC gateway endpoint for an AWS Simple Storage Service (S3).
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS infrastructure, you must review the provided information and manually create the infrastructure. If your cluster does not initialize correctly, you might have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* You configured an AWS account.
+* You added your AWS keys and region to your local AWS profile by running `aws configure`.
+
+.Procedure
+
+.  Go to the next section of the documentation named "CloudFormation template for the VPC Carrier Gateway", and then copy the syntax from the *CloudFormation template for VPC Carrier Gateway* template. Save the copied template syntax as a YAML file on your local system. This template describes the VPC that your cluster requires.
+
+. Run the following command to deploy the CloudFormation template, which creates a stack of AWS resources that represent the VPC:
++
+[IMPORTANT]
+====
+You must enter the command on a single line.
+====
```

Comment:
Any reason for this comment/requirement/admotion?

Reply:
I cannot easily determine the origin given the size of the PR. Note removed.